web/src/testing/render.tsx

```diff
@@ -17,14 +17,14 @@ type RenderRouteOptions = {
 
 const theme = { ...buildConfig().lightTheme, contentDirection: 'ltr' as UiDirectionType }
 
-const AllTheProviders = ({ children, options }: { children: ReactElement; options?: { pathname: string } }) => (
+const AllTheProviders = ({ children, options }: { children: ReactNode; options?: { pathname: string } }) => (
   <MemoryRouter initialEntries={options ? [options.pathname] : ['/']}>
     <ThemeProvider theme={theme}>{children}</ThemeProvider>
   </MemoryRouter>
 )
 
 export const renderWithRouterAndTheme = (ui: ReactElement, options?: { pathname: string }): RenderResult =>
-  render(ui, { wrapper: (props: { children: ReactElement }) => <AllTheProviders {...props} options={options} /> })
+  render(ui, { wrapper: (props: { children: ReactNode }) => <AllTheProviders {...props} options={options} /> })
```

Comment:
Why would we allow ReactNodes here? "ReactElements" make more sense, don't they?

Reply:
This is the type required for @testing-library and I could not get it to pass type checking otherwise. But feel free to come up with another solution. I think this is very minor priority since its only tests anyway...